partiql-ast/build.gradle.kts

```diff
@@ -55,24 +55,3 @@ publish {
     name = "PartiQL AST"
     description = "PartiQL's Abstract Syntax Tree"
 }
-
-val generate = tasks.register<Exec>("generate") {
```

Comment:
(self-review) remove sprout-generation from AST build

Reply:
This is huge, thank you so much!